cmd/podman/inspect/inspect.go

```diff
@@ -2,7 +2,7 @@ package inspect
 
 import (
 	"context"
-	"encoding/json" // due to a bug in json-iterator it cannot be used here
+	"encoding/json"
```

Comment:
Why remove comment? Anyone who tries to use json-iterator (like most of cmd/podman) will have to debug the issue again...